### ISSUE TITLE
Apply the `fs_inotify` crate to `c8y_log_plugin`

### ISSUE THREAD

**Issue by rina23q** [thin-edge/thin-edge.io#1247]
After https://github.com/thin-edge/thin-edge.io/issues/1173 is done, we need to apply the crate to `c8y_log_plugin` to detect the changes on configuration file of the plugin..

AC:
- `c8y_log_plugin` consumes the fs_inotify crate
- Tested

**Comment by gligorisaev** [comment]
Missing information how this issue is resolved

**Comment by initard** [comment]
# How to Test

```gherkin
Feature: changes to log plugin are automatically updated in the cloud

    @important
    Scenario Outline: changing log file 
      Given DUT has c8y-log-plugin installed
      And DUT is connected to c8y
      When the log file is changed
      Then the log is updated in the cloud

      Extra information:
          The log file is located at: /etc/tedge/c8y/c8y-log-plugin.toml.
          Changed log file example:
          files = [
              { type = "software-management", path = "/var/log/tedge/agent/software-*" },
              { type = "mosquitto", path = "/var/log/mosquitto/mosquitto.log" }
          ]
          These entries should be seen on the cloud-side in the Logs tab.
```

**Comment by rina23q** [comment]
Resolved by #1245
